src/components/view/DeviceActivityView/DeviceActivityView.tsx

```diff
@@ -24,7 +24,7 @@ export default function DeviceActivityView(props: DeviceActivityViewProps) {
             showCloseButton={props.presentation == "Modal"}
             colorScheme={props.colorScheme}>
             <SegmentedControl
-                style={{ marginLeft: "16px", marginRight: "16px", marginBottom: selectedView == "daily" ? "16px" : "0px" }}
+                style={{ marginTop: !props.presentation ? "16px" : "0px", marginLeft: "16px", marginRight: "16px", marginBottom: selectedView == "daily" ? "16px" : "0px" }}
```

Comment:
This also adds a small tweak to DeviceActivityView so the segmented control isn't pushed up against the top of the screen if there's no "presentation" arg